lib/maril/generator.rb

```diff
@@ -19,7 +19,7 @@ def generate
       end
       cmd << app['container']['docker']['image']
       cmd << app['cmd'] if app['cmd']
-      cmd.join(" \\\n")
+      cmd.join(" ")
```

Comment:
This is the only one thing that I don't agree. I prefer to split options to different lines and use `\` at the end of the line. It can make the entire command easier to be read.